google/cloud/bigquery/reservation_v1/services/reservation_service/client.py

```diff
@@ -1823,6 +1823,7 @@ def search_assignments(
                 -  ``assignee=organizations/456``
 
 
+
```

Comment:
Oops. I forgot to remove these files. Updated in https://github.com/googleapis/python-bigquery-reservation/pull/52